doc/contributing/adding-new-napi-api.md

```diff
@@ -37,7 +37,7 @@ please follow these principles and guidelines:
 
 2. **Review and approval**
    * A new API addition **must** be signed off by at least two Node-API team members.
-   * **Should** be implemented in at least one other VM implementation of Node.js.
+   * **Should** be implementable in at least one other VM implementation of Node.js.
```

Comment:
"implementable" is very vague, even weasel wordy. Everything is implementable given enough time and effort.

Reductio ad ad absurdum: parsing and executing Python code isn't implemented in V8 but given a ground-up rewrite sure is implementable. Surely that's not what you mean though.

"Can be implemented in terms of available VM APIs" maybe?

Reply:
Thanks for suggestion, updated!